doc/classes/EditorSelection.xml

```diff
@@ -30,10 +30,18 @@
 				Returns the list of selected nodes.
 			</description>
 		</method>
-		<method name="get_transformable_selected_nodes">
+		<method name="get_top_selected_nodes">
 			<return type="Node[]" />
 			<description>
-				Returns the list of selected nodes, optimized for transform operations (i.e. moving them, rotating, etc.). This list can be used to avoid situations where a node is selected and is also a child/grandchild.
+				Returns the list of top selected nodes only, excluding any children. This is useful for performing transform operations (moving them, rotating, etc).
```

Comment:
> Returns the list of top selected nodes only, excluding any children.

This doesn't properly convey what are "top nodes". Is it e.g. root node? Does it mean any child node is excluded, even if their parent is not selected?

Reply:
That's why I specified "top selected nodes", so it's the nodes on top in the list of selected nodes. I'm open to suggestions on making this clearer.

Reply:
CC @Mickeon 

Also nitpick:
```suggestion
				Returns the list of top selected nodes only, excluding any children. This is useful for performing transform operations (moving them, rotating, etc.).
```
Same below.